.env.example

```diff
@@ -1,7 +1,15 @@
 # Environment variables for browser-calls-flask
 
+# The SECRET_KEY can be anything. For more information: https://flask.palletsprojects.com/en/2.0.x/config/#SECRET_KEY
 SECRET_KEY=
 
+# If you'd prefer to use your own database, use this environment variable
```

Comment:
Maybe specify that you also have to set FLASK_ENV to "production" to use your own database/not sqllite. (You specify it below too, but maybe for extra clarity, because just setting this variable won't work).